### DISCUSSION TITLE
PrismaORM with multiple data sources

### DISCUSSION THREAD

**Discussion by rdrigos** [prisma/prisma#24243] [Q&A]
### Question

How can I use PrismaORM with multiple data sources? I mean, two different connections:

Connection 1 (A connection string and a schema)
Connection 1 (A connection string and a schema)

**Comment by ludralph** [comment]
Hi @rdrigos 👋 

You can take a look at this [comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) for how to use multiple data sources with Prisma.

**Reply by rdrigos** [reply]
OK thank you! I will mark this as completed to centralize the information in a single post.
